src/Tests/Microsoft.NET.Build.Tests/WorkloadTests.cs

```diff
@@ -237,5 +237,67 @@ public void It_should_import_aliased_pack()
                 .Should()
                 .BeEquivalentTo("true");
         }
+
+        [Fact]
+        public void It_should_get_suggested_workload_by_FindAllReferenceWorkload_target()
+        {
+            var mainProject = new TestProject()
+            {
+                Name = "MainProject", TargetFrameworks = "net6.0-android", IsSdkProject = true, IsExe = true
+            };
+
+            var testAsset = _testAssetsManager
+                .CreateTestProject(mainProject);
+
+            var getValuesCommand =
+                new GetValuesCommand(testAsset, "_ResolvedSuggestedWorkloads", GetValuesCommand.ValueType.Item);
+            getValuesCommand.DependsOnTargets = "_FindAllReferenceWorkload";
+            getValuesCommand.ShouldRestore = false;
+
+            getValuesCommand.Execute("/p:SkipResolvePackageAssets=true")
+                .Should()
+                .Pass();
+
+            getValuesCommand.GetValues()
+                .Should()
+                .BeEquivalentTo("microsoft-android-sdk-full", "microsoft-net-runtime-mono-tooling");
```

Comment:
I think these names are changing, you may want to update stage 0 and see if that includes the rename so that this test doesn't start failing when it happens.

Reply:
due to https://github.com/dotnet/sdk/issues/19021 I will update stage 0 in a different PR